nixos/modules/services/networking/searx.nix

```
@@ -90,12 +89,12 @@ in
         example = literalExpression ''
           { server.port = 8080;
             server.bind_address = "0.0.0.0";
-            server.secret_key = "@SEARX_SECRET_KEY@";
+            server.secret_key = "$\{SECRET_KEY}";
```

```suggestion
            server.secret_key = "''${SECRET_KEY}";
```
In multiline strings, ${} is escaped with two apostrophes. Same below

---

Okay thank you